### PR TITLE
{lib}[GCCcore/11.3.0] Unidecode v1.3.6 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3113.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/61341cd2b132c79817d191526c5d5013 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/e67e1219ebe0d8cfd78f7fe24ba6a0f8 for a full test report.

File: easybuild/easyconfigs/u/Unidecode/Unidecode-1.3.6-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
```
Comment (verdurin) [inline easybuild/easyconfigs/u/Unidecode/Unidecode-1.3.6-GCCcore-11.3.0.eb:1]: Strictly speaking this should be a `PythonBundle` now?

Comment (verdurin) [reply easybuild/easyconfigs/u/Unidecode/Unidecode-1.3.6-GCCcore-11.3.0.eb:1]: It's fine as is for now.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**